src/module-elasticsuite-core/Setup/ConfigOptionsList.php

```diff
@@ -80,8 +80,8 @@ public function __construct(
         SearchConfigOptionsList $searchConfigOptionsList,
         $fallbackMapping = []
     ) {
-        $this->clientBuilder           = $clientBuilder;
-        $this->fallbackMapping         = array_merge($this->fallbackMapping, $fallbackMapping);
+        $this->clientBuilder = $clientBuilder;
```

Comment:
Don't change this cosmetic, we like things aligned, and changing these lines is not related to your PR.